src/test/java/software/amazon/neptune/csv2rdf/PropertyGraph2RdfMappingTest.java

```diff
@@ -320,4 +321,16 @@ public void invalidPatternIsRejected() {
 				exception.getMessage());
 	}
 
+	@Test
+	public void testGetEdgeContextNamespace() {
```

Comment:
public void testGetEdgeContextNamespaceFallback() {
```suggestion
	public void testGetEdgeContextNamespaceFallback() {
```